src/test/resources/org/broadinstitute/hellbender/tools/haplotypecaller/expected.pileupCallerDRAGEN.gatk4.vcf

```diff
@@ -0,0 +1,148 @@
+20	10093923	.	T	A	1096.64	.	AC=1;AF=0.500;AN=2;BaseQRankSum=-0.667;DP=73;ExcessHet=3.0103;FS=0.924;MLEAC=1;MLEAF=0.500;MQ=59.25;MQRankSum=0.358;QD=16.37;ReadPosRankSum=-1.583;SOR=0.772	GT:AD:DP:GQ:PL	0/1:35,32:67:99:1104,0,1257
+20	10094251	.	T	A	1291.64	.	AC=1;AF=0.500;AN=2;BaseQRankSum=0.251;DP=67;ExcessHet=3.0103;FS=11.483;MLEAC=1;MLEAF=0.500;MQ=60.00;MQRankSum=0.000;QD=20.83;ReadPosRankSum=1.806;SOR=1.211	GT:AD:DP:GQ:PL	0/1:26,36:62:99:1299,0,781
+20	10095741	.	A	G	1096.64	.	AC=1;AF=0.500;AN=2;BaseQRankSum=-0.534;DP=72;ExcessHet=3.0103;FS=4.721;MLEAC=1;MLEAF=0.500;MQ=57.39;MQRankSum=-2.313;QD=15.67;ReadPosRankSum=-0.500;SOR=0.515	GT:AD:DP:GQ:PL	0/1:36,34:70:99:1104,0,1233
+20	10096768	.	A	C	1703.64	.	AC=1;AF=0.500;AN=2;BaseQRankSum=-2.764;DP=96;ExcessHet=3.0103;FS=3.983;MLEAC=1;MLEAF=0.500;MQ=59.28;MQRankSum=-1.609;QD=18.72;ReadPosRankSum=0.613;SOR=0.445	GT:AD:DP:GQ:PL	0/1:42,49:91:99:1711,0,1452
+20	10096958	.	G	A	566.64	.	AC=1;AF=0.500;AN=2;BaseQRankSum=1.277;DP=64;ExcessHet=3.0103;FS=7.479;MLEAC=1;MLEAF=0.500;MQ=60.00;MQRankSum=0.000;QD=8.99;ReadPosRankSum=-2.151;SOR=0.784	GT:AD:DP:GQ:PL	0/1:44,19:63:99:574,0,1571
+20	10097075	.	T	G	1440.64	.	AC=1;AF=0.500;AN=2;BaseQRankSum=-2.398;DP=95;ExcessHet=3.0103;FS=6.783;MLEAC=1;MLEAF=0.500;MQ=58.12;MQRankSum=-1.398;QD=15.83;ReadPosRankSum=0.242;SOR=0.694	GT:AD:DP:GQ:PL	0/1:44,47:91:99:1448,0,1485
+20	10097789	.	T	C	709.64	.	AC=1;AF=0.500;AN=2;BaseQRankSum=-1.157;DP=60;ExcessHet=3.0103;FS=5.374;MLEAC=1;MLEAF=0.500;MQ=58.05;MQRankSum=-2.805;QD=12.45;ReadPosRankSum=0.220;SOR=0.756	GT:AD:DP:GQ:PL	0/1:34,23:57:99:717,0,1190
+20	10097928	.	G	A	591.64	.	AC=1;AF=0.500;AN=2;BaseQRankSum=-1.203;DP=37;ExcessHet=3.0103;FS=1.315;MLEAC=1;MLEAF=0.500;MQ=58.11;MQRankSum=-1.654;QD=16.43;ReadPosRankSum=-0.079;SOR=0.951	GT:AD:DP:GQ:PL	0/1:17,19:36:99:599,0,586
+20	10098110	.	G	C	573.64	.	AC=1;AF=0.500;AN=2;BaseQRankSum=2.510;DP=36;ExcessHet=3.0103;FS=6.658;MLEAC=1;MLEAF=0.500;MQ=58.71;MQRankSum=-1.477;QD=15.93;ReadPosRankSum=-0.730;SOR=0.069	GT:AD:DP:GQ:PL	0/1:19,17:36:99:581,0,631
+20	10098786	.	C	T	32.64	.	AC=1;AF=0.500;AN=2;BaseQRankSum=0.431;DP=9;ExcessHet=3.0103;FS=0.000;MLEAC=1;MLEAF=0.500;MQ=54.65;MQRankSum=-2.200;QD=3.63;ReadPosRankSum=-1.383;SOR=0.132	GT:AD:DP:GQ:PL	0/1:7,2:9:40:40,0,217
+20	10099220	.	A	G	598.64	.	AC=1;AF=0.500;AN=2;BaseQRankSum=-1.664;DP=47;ExcessHet=3.0103;FS=1.137;MLEAC=1;MLEAF=0.500;MQ=57.74;MQRankSum=-2.596;QD=13.30;ReadPosRankSum=0.629;SOR=0.519	GT:AD:DP:GQ:PL	0/1:25,20:45:99:606,0,873
+20	10099535	.	G	A	1356.64	.	AC=1;AF=0.500;AN=2;BaseQRankSum=5.034;DP=68;ExcessHet=3.0103;FS=2.172;MLEAC=1;MLEAF=0.500;MQ=59.04;MQRankSum=-1.418;QD=20.87;ReadPosRankSum=-0.777;SOR=0.408	GT:AD:DP:GQ:PL	0/1:26,39:65:99:1364,0,686
+20	10099565	.	C	T	1367.64	.	AC=1;AF=0.500;AN=2;BaseQRankSum=4.595;DP=70;ExcessHet=3.0103;FS=8.736;MLEAC=1;MLEAF=0.500;MQ=59.40;MQRankSum=-1.266;QD=19.82;ReadPosRankSum=1.225;SOR=0.191	GT:AD:DP:GQ:PL	0/1:31,38:69:99:1375,0,994
```

Comment:
there's a utuilty method that indexes vcfs for testing, perhaps better to use it than to include indexes in the repository?